app/components/explorer/download-data-dropdown.js

```diff
@@ -0,0 +1,135 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+
+/**
+  * @param { Object[] } topics - array of Topic objects.
+  * If explorer selected source is acs, then `topics` has
+  * TWO levels of nesting of Topic objects (Topics and Subtopics).
+  * If explorer selected source is decennial, then `topics` is a
+  * 1D array of subtopics.
+  * See controller for an example.
+*/
+
+export default class DownloadDataDropdown extends Component{
+  @tracked open = false;
+
+  @action toggleOpen() {
+    this.open = !this.open;
+  }
+
+  @action downloadCSV(csv) {
+    var csvFile;
+    var downloadLink;
+    const d = new Date(Date.now());
+    var filename = ['popfactfinder.planning.nyc.gov_data', d.getFullYear(), ("0" + (d.getMonth() + 1)).slice(-2), ("0" + d.getDate()).slice(-2) ].join('-').concat('.csv');
+
+    // CSV file
+    csvFile = new Blob([csv], {type: "text/csv"});
+
+    // Download link
+    downloadLink = document.createElement("a");
+
+    // File name
+    downloadLink.download = filename;
+
+    // Create a link to the file
+    downloadLink.href = window.URL.createObjectURL(csvFile);
+
+    // Hide download link
+    downloadLink.style.display = "none";
+
+    // Add the link to DOM
+    document.body.appendChild(downloadLink);
+
+    // Click download link
+    downloadLink.click();
+  }
+  @action convertStringForCSV(s) {
+    if(s.indexOf(',') !== -1) {
+      return '"'.concat(s).concat('"');
+    } 
+    if(s===" ") {
+      return "";
+    }
+    return s;
+  }
+  @action async exportTableToCSV() {
+    // This toggles the reliability data on if it is not before downloading
+    const initialReliabilitySetting = this.args.showReliability;
+    if(!initialReliabilitySetting) { await this.args.toggleReliability(); }
```

Comment:
style nit: can you break these if statement blocks out into their own line? and also add spaces around the conditional? Similar to the lint styles for the backend